catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -1021,27 +999,59 @@ public UpdateResponse update(UpdateRequest updateRequest)
 
         UpdateResponse updateResponse = null;
         try {
+            List<Filter> idFilters = new ArrayList<>();
```

Comment:
Most of the changes in this file are just reformatting. Look here for changes, and in the delete method as well.